codex-rs/azureapi.md

```
@@ -0,0 +1,1236 @@
+file = client.files.create(
+  file=open("nucleus_sampling.pdf", "rb"), # This assumes a .pdf file in the same directory as the executing script
+  purpose="assistants"
+)
+
+print(file.model_dump_json(indent=2))
+file_id = file.id
+```
+
+**Output:**
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)